linux/.gitignore

```diff
@@ -1 +1,2 @@
 flutter/ephemeral
+flutter/generated*
```

Comment:
I was getting really tired of having to deal with version control for generated files.